UnityProject/Assets/Scripts/ChemistryComponents/ReagentContainer.cs

```diff
@@ -134,6 +135,16 @@ public ReagentMix CurrentReagentMix
 			}
 		}
 
+		//[RightClickMethod]
+		public void PrintContents()
+		{
+			foreach (var keyd in currentReagentMix.reagents.m_dict)
+			{
+				Logger.LogError(keyd.Key.Name + " > " + keyd.Value);
+			}
+			Logger.LogError("=====================================");
```

Comment:
Why an error log for a print function? I am 75.6% sure we have something like just a normal Logger.Log() that's not as an error.

Reply:
is a debug, also easy to see in the Consul

Reply:
It isn't an error. Don't log non errors as errors. You can do it while you debug locally, but don't push it to prod